### PR TITLE
Audio recordings update alert actions

### PR DESCRIPTION
**Pull request by JB184351**

### CODE REVIEW

File: TalkAloud/TalkAloud/ViewControllers/AudioRecordingsTableViewController.swift [hunk 1]
```diff
@@ -48,16 +48,27 @@ class AudioRecordingsTableViewController: UITableViewController {
...
-            AudioManager.sharedInstance.removeFile(at: indexPath.row)
-            tableView.deleteRows(at: [indexPath], with: .automatic)
+            
+            let deleteAlertController = UIAlertController(title: "Are you sure you want to delete?", message: "You won't be able to recover this file", preferredStyle: .alert)
+            let deleteAlertAction = UIAlertAction(title: "Delete", style: .destructive, handler:  { _ in
+                AudioManager.sharedInstance.removeFile(at: indexPath.row)
+                tableView.deleteRows(at: [indexPath], with: .automatic)
+            })
+            let cancelDeleteAction = UIAlertAction(title: "Cancel", style: .cancel, handler: { _ in
+                completionHandler(false)
```
Comment (virgil-adumitroae) [inline TalkAloud/TalkAloud/ViewControllers/AudioRecordingsTableViewController.swift:58]: You can leave the completion block empty here or mark it as nil

File: TalkAloud/TalkAloud/ViewControllers/AudioRecordingsTableViewController.swift [hunk 2]
```diff
@@ -73,10 +84,16 @@ class AudioRecordingsTableViewController: UITableViewController {
                     
                     self.tableView.reloadData()
                 }
+                
             }
             
-            ac.addAction(renameFileAction)
-            self.present(ac, animated: true)
+            let cancelEditAction = UIAlertAction(title: "Cancel", style: .cancel, handler: { _ in
+                completionHandler(false)
```
Comment (virgil-adumitroae) [inline TalkAloud/TalkAloud/ViewControllers/AudioRecordingsTableViewController.swift:91]: You can leave the completion block empty here or mark it as nil

**Review by virgil-adumitroae — CHANGES_REQUESTED** [review]

File: TalkAloud/TalkAloud/ViewControllers/AudioRecordingsTableViewController.swift (hunk 1 above)
Comment (JB184351) [reply TalkAloud/TalkAloud/ViewControllers/AudioRecordingsTableViewController.swift:58]: Would I still be able to set the completionHandler as false? I needed this to get rid of the UIConextualAction UI after selecting cancel.

File: TalkAloud/TalkAloud/ViewControllers/AudioRecordingsTableViewController.swift (hunk 2 above)
Comment (JB184351) [reply TalkAloud/TalkAloud/ViewControllers/AudioRecordingsTableViewController.swift:91]: Would I still be able to set the completionHandler as false? I needed this to get rid of the UIConextualAction UI after selecting cancel.

Comment (virgil-adumitroae) [reply TalkAloud/TalkAloud/ViewControllers/AudioRecordingsTableViewController.swift:91]: Ah ok makes sense then :D

**Pull request merged**